### DISCUSSION TITLE
emqx_elixir_plugin prints undesired payloads

### DISCUSSION THREAD

**Discussion by SSaiPrasad7** [emqx/emqx#4429] [Q&A] [answered]
![emqx-bug](https://user-images.githubusercontent.com/36000620/112303861-ce712e00-8cc2-11eb-93c6-9d5de764e585.png)
Hello @ zmstone
When I am using `emqx_elixir_plugin` I observed that unwanted values are displayed on console. I am sending data from a range about (10,50) but it is displaying out of range values.
Please check this issue before adding to enterprise version.
Thank you and All the best.

**Comment by zmstone** [comment, marked as answer]
these are payloads from `$SYS` prefixed topics.

the default elixir plugin just dumps everything to console (topic is not printed)
you probably need to inspect the topics and do a bit of filtering
